src/features/auth/components/AuthForm/AuthForm.css

```diff
@@ -50,26 +49,37 @@
 }
 
 .auth-form__button {
+  position: relative;
   margin: 0 auto;
+  outline: none;
   border: none;
-  border-radius: 30px;
-  padding: 16px;
-  min-width: 193px;
+  border-radius: 15px;
+  padding: 10px 42px;
+  min-width: 137px;
   box-shadow: none;
   font-size: 18px;
   line-height: 1.15;
-  color: #fff;
-  background-color: #349abc;
+  color: #172f3d;
```

Comment:
color could be `inherit`
and border-color could be `currentColor`

because this button uses same colors with text by design, as i can see